### ISSUE TITLE
Should I really work on this thing?

### ISSUE THREAD

**Issue by Hazuki-san** [Hazuki-san/hqOsu-xtreme#1]
I know I've been stopped working on everything for a while... but after thinking all of that...

My personal stuff/work has been leaked almost like everywhere (because they're probably not *kinda* happy that I didn't release anything to public).

There's a reason that I've stopped working on this
- There's an unofficial forks that tried to remove a anticheat in my cheat (that is limiting my OWN osu! server and others if they're friendly to me or contacted me).
- They're trying to claim my work is their's
- I've recieved too many messages about updates
- Not worth doing it because the cheat is old

So here's what I'm thinking... Should I really work on this thing again? Will people stop doing those thing I mentioned above? Is it really worth it to update it again?

Let me know your comments.
~ @Hazuki-san

**Comment by sqlskid** [comment]
please work on this <3 we all love you

**Comment by Bumply** [comment]
Well you know those people are always gonna mess with you I want you to continue couse I respect your work but it's up to you because it's not gonna be easy

**Comment by sezamek** [comment]
To be honest, I don't think It's worth to continue this. There will always be people, who will fork that and make a mess from it. Imo, hq should be graved. I appreciate your work, maybe some too. But let hq-osu die...

**Comment by LightCyan01** [comment]
You don't need to do this, but instead focus on your client. I'm positive you can protect your client better than doing hq. If you decide to do this, you can add extra steps like verifications and maybe a login before being able to use this. I have a suggestion which is to create a loader application that after logging into the loader, hq.osu gets downloaded from server to loader and loads from there. Then deletes itself after using. That way people will not be able to crack it easily, also you can fully encrypt your loader.  If I'm saying anything wrong don't bully me.

**Comment by oSumAtrIX** [comment]
> You don't need to do this, but instead focus on your client. I'm positive you can protect your client better than doing hq. If you decide to do this, you can add extra steps like verifications and maybe a login before being able to use this. I have a suggestion which is to create a loader application that after logging into the loader, hq.osu gets downloaded from server to loader and loads from there. Then deletes itself after using. That way people will not be able to crack it easily, also you can fully encrypt your loader. If I'm saying anything wrong don't bully me.

After the loader loads the file one can simply dump it.

**Comment by venosu** [comment]
Yes please, so mean people count on you

**Comment by kird3333d** [comment]
for me going back to hqosu would be good because a lot of people prefer extrenal cheat to modded client and for me it doesn't matter what you do anyway because your projects are good and I love them

**Comment by rose-tinted-glasses** [comment]
regardless of what cheat it is, you're always going to have someone fork it and make it their own. you can't call this cheat your own when it's a fork of someone else's work either. you expecting this cheating community to not do what you're describing is simply unrealistic, and if that's a problem for you then you should not continue it, especially since it's open to the public on github.

**Comment by rose-tinted-glasses** [comment]
and besides, if someone really wanted to use hqosu -- they could just use the legacy fork cherry made and change the version number in config.ini, and it will work fine with no issues

**Comment by Hazuki-san** [comment]
> and besides, if someone really wanted to use hqosu -- they could just use the legacy fork cherry made and change the version number in config.ini, and it will work fine with no issues

This is wrong, it is not working like that. The legacy is based on 2019 version which is very different from current osu! today. Current osu! has an anticheat called osu!auth.dll, and legacy doesn't have it. If you attempt to use it there won't be "Token" header and that'll flag your account. Another thing 2019 version doesn't have the same the flag same as 2021 client. So using that will flagging your account and may come as instant restriction.

**Comment by rose-tinted-glasses** [comment]
> > and besides, if someone really wanted to use hqosu -- they could just use the legacy fork cherry made and change the version number in config.ini, and it will work fine with no issues
> 
> This is wrong, it is not working like that. The legacy is based on 2019 version which is very different from current osu! today. Current osu! has an anticheat called osu!auth.dll, and legacy doesn't have it. If you attempt to use it there won't be "Token" header and that'll flag your account. Another thing 2019 version doesn't have the same the flag same as 2021 client. So using that will flagging your account and may come as instant restriction.

i use legacy on another account with no restrictions on latest, so i’m speaking from experience and not technically — it probably has flagged my account but at this point what good is flagging my account doing when i’ve been unrestricted for months at this point

**Comment by xxkat** [comment]
don't bother

**Comment by Sockings** [comment]
> > > and besides, if someone really wanted to use hqosu -- they could just use the legacy fork cherry made and change the version number in config.ini, and it will work fine with no issues
> > 
> > 
> > This is wrong, it is not working like that. The legacy is based on 2019 version which is very different from current osu! today. Current osu! has an anticheat called osu!auth.dll, and legacy doesn't have it. If you attempt to use it there won't be "Token" header and that'll flag your account. Another thing 2019 version doesn't have the same the flag same as 2021 client. So using that will flagging your account and may come as instant restriction.
> 
> i use legacy on another account with no restrictions on latest, so i’m speaking from experience and not technically — it probably has flagged my account but at this point what good is flagging my account doing when i’ve been unrestricted for months at this point

I'm pretty sure past a point your account gets restricted, I mostly play multi but I haven't been able to play for the past months since I either get banned after hitting 200k or some kid whos friends with a mod reports me and gets me restricted . . .

SO I only really play offline just for the fun of it!
